tests/integration/test_model.py

```diff
@@ -924,6 +925,7 @@ def test_experiment_automl_classifier(c, client, training_df):
 
 # TODO - many ML tests fail on clusters without sklearn - can we avoid this?
 @skip_if_external_scheduler
+@pytest.mark.skip(reason="Waiting on https://github.com/EpistasisLab/tpot/pull/1280")
```

Comment:
TPOT fails with the newest NumPy version as well. I've opened https://github.com/EpistasisLab/tpot/pull/1280 to resolve this.